RxSwift/Concurrency/Lock.swift

```diff
@@ -16,7 +16,7 @@ typealias SpinLock = RecursiveLock
 
 extension RecursiveLock : Lock {
     @inline(__always)
-    final func performLocked<T>(_ action: () -> T) -> T {
+    final func performLocked<Element>(_ action: () -> Element) -> Element {
```

Comment:
I think `T` fits fine here since it's not an element, it's just some generic value

Reply:
Kept it as <T> instead of <Element> as suggested!

Reply:
But I just have a question. Isn't it much better if we put `<element>` which is more descriptive than `<T>`? 
Just curious which one would be better. Can you share your opinion?

Reply:
Hey @jubakong - 
We use the `Element` terminology for Observables since they emit elements. It doesn't really fit here because performLocked can return any value, and I think `T` is a fine placeholder. If you want to be explicit you can use `Value`, instead.